promql/query_logger_test.go

```diff
@@ -111,7 +111,10 @@ func TestMMapFile(t *testing.T) {
 	filename := file.Name()
 	defer os.Remove(filename)
 
-	fileAsBytes, err := getMMapedFile(filename, 2, nil)
+	fileAsBytes, closer, err := getMMapedFile(filename, 2, nil)
+	if err != nil {
+		t.Cleanup(func() { closer.Close() })
```

Comment:
```suggestion
		t.Cleanup(closer.Close)
```

Reply:
I wanted to write as this way at first. But `io.Closter` is `Close() error`, cannot use in `testing.Cleanup(f func())`

Reply:
` cannot use closer.Close (value of type func() error) as func() value in argument to t.Cleanup`. so, it seems have to embase `closer.Close` into a function